src/libraries/System.Net.Mail/ref/System.Net.Mail.cs

```diff
@@ -165,36 +165,65 @@ public enum MailPriority
         High = 2,
     }
     public delegate void SendCompletedEventHandler(object sender, System.ComponentModel.AsyncCompletedEventArgs e);
+    [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
```

Comment:
Though it doesn't logically affect anything (I think) by having the Unsupported attribute at both the class level and method level, it feels unnecessary

Reply:
I think it would be better to keep the marking at the class level.

Reply:
Just wanted to confirm.  Removed them.  Thanks